### ISSUE TITLE
Setting of scheduled enqueue time via Message.schedule is not reflected in Message.scheduled_enqueue_time

### ISSUE THREAD

**Issue by zuckerruebe** [Azure/azure-sdk-for-python#12861]
- **Package Name**: 
azure-servicebus
- **Package Version**: 
0.50.3
- **Operating System**:
Windows 10
- **Python Version**: 
3.7.7

**Describe the bug**
Setting of scheduled enqueue time via `azure.servicebus.Message.schedule` is not reflected in `azure.servicebus.Message.scheduled_enqueue_time`.

**To Reproduce**
```
import azure.servicebus as sb
message = sb.Message('hi there!')
import datetime as dt
message.schedule(dt.datetime.now(dt.timezone.utc)+dt.timedelta(minutes=5))
message.scheduled_enqueue_time
```

**Expected behavior**
`azure.servicebus.Message.scheduled_enqueue_time` should reflect the time set using 
`azure.servicebus.Message.schedule`.

**Comment by lmazuel** [comment]
Thanks @zuckerruebe ! Our ServiceBus expert will come soon to investigate your question ;)

**Comment by yunhaoling** [comment]
hey @zuckerruebe , thanks for the report.

The root cause for this issue is that `Message` object serves as both incoming and outgoing message; however the key we used to set/get the schedule time information is not set property depending on the incoming/outgoing type.

We've been aware of the issue and will get it fixed in our next servicebus preview release 7.0.0b5. We will let you know when it's out so that you could take a try.

FYI: The v0.50 servicebus library has been deprecated now, please migrate to our latest v7 servicebus library following the [migration guide](https://github.com/Azure/azure-sdk-for-python/blob/master/sdk/servicebus/azure-servicebus/migration_guide.md).

Thanks for you understanding and patience in advance!

**Comment by yunhaoling** [comment]
hey @zuckerruebe , [azure-servicebus 7.0.0b5](https://pypi.org/project/azure-servicebus/7.0.0b5/) is released today which have addressed the issue.
Please note that `scheduled_enqueue_time` is now renamed as `scheduled_enqueue_time_utc`.

I'm closing this issue. Feel free to reopen if you still encounter the problem.
